azkaban-db/src/main/java/azkaban/db/DatabaseTransOperator.java

```diff
@@ -30,14 +30,14 @@
  */
 public interface DatabaseTransOperator {
```

Comment:
Curious: What is the difference between `DatabaseOperator` and `DatabaseTransOperator`?